### PR TITLE
WW-89: Checking if data returned from cache is an array

### PR DESCRIPTION
**Pull request by rybmat**
Ticket: https://wikia-inc.atlassian.net/browse/WW-89

Error was occurred because "get" method from cache returns false if there is no data for given key.

@Wikia/west-wing

### CODE REVIEW

**Comment by sebastianmarzjan** [comment]
LGTM

File: extensions/wikia/InsightsV2/helpers/InsightsCache.php [hunk 1]
```diff
@@ -17,7 +17,8 @@ public function __construct( InsightsConfig $config ) {
 	}
 
 	public function get( $params ) {
-		return $this->memc->get( $this->getMemcKey( $params ) );
+		$data = $this->memc->get( $this->getMemcKey( $params ) );
+		return is_array($data) ? $data : [];
```
Comment (lukaszkonieczny) [inline extensions/wikia/InsightsV2/helpers/InsightsCache.php]: Please add spaces inside brackets per convention

**Comment by lukaszkonieczny** [comment]
I think after this change we should change condition here https://github.com/Wikia/app/blob/e5aa9665b7b2122cbb6c602b8e7dd39869f79653/extensions/wikia/InsightsV2/helpers/InsightsCache.php#L86

**Comment by sebastianmarzjan** [comment]
'change' as in 'remove'?

**Comment by lukaszkonieczny** [comment]
I will check `!empty()`

**Comment by rybmat** [comment]
good catch!

**Comment by sebastianmarzjan** [comment]
👍

**Pull request merged**